plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -123,6 +161,16 @@ const CatalogPageContents = () => {
       <Content>
         <WelcomeBanner />
         <ContentHeader title={selectedTab ?? ''}>
+          {entitiesState && entitiesState.length === 0 ? (
+            <Button
+              onClick={addMockData}
+              variant="contained"
+              color="primary"
+              className={styles.mockDataButton}
+            >
```

Comment:
Is there a way to put the button into the table where it says "No records to display". The location would make it slightly more obvious what was going to happen.

Reply:
I agree with @dtuite -- that would be even better.

Reply:
Yup, that would be better. Give me some time to explore and try it out